app/models/hackathon.rb

```diff
@@ -2,12 +2,13 @@ class Hackathon < ApplicationRecord
   include Eventable
   include Taggable
 
-  include Status
-  include Named
-  include Regional
-  include Scheduled
   include Applicant
   include Brand
+  include FinanciallyAssisting # depends on Taggable
```

Comment:
```suggestion
  include FinanicalAssistance # depends on Taggable
```

Reply:
It's not a completely separate entity, it's more an aspect of a `Hackathon`. "Lemme tell you about `Hackathon`s! They're `Scheduled`, might be `FinanciallyAssisting`..." - it's structured writing.